.github/workflows/my-ci-pipeline.yml

```diff
@@ -74,5 +74,8 @@ jobs:
       - name: Azure Update
         run: curl -dH -X POST '${{ secrets.AZURE_HOOK_TODO }}'
 
-      - name: Teraform
+      - name: Teraform initialize the working directory
         run: terraform init
+
+      - name: Apply teraform changes
+        run: terraform apply
```

Comment:
This definitely won't be enough to work in isolation!
I'm expecting to see:
* Secrets loaded into Environment variables for `ARM_CLIENT_ID, ARM_CLIENT_SECRET, ARM_TENANT_ID, ARM_SUBSCRIPTION_ID`
* Secrets/variables for each of your non-default input variables
* `-auto-approve`, otherwise TF will just sit here and spin waiting for confirmation
* The `curl` command wants to execute against the terraform output so that we don't need to specify the Hook address directly

Reply:
are you free for a call this week, I have a few questions about this?